book/invalidation.md

```diff
@@ -2077,7 +2066,7 @@ everything! I won't implement it, but paint can be made a lot faster
 too---see Exercise 16-10.)
 
 [^other-phases]:  It might also be pretty laggy on large pages due to the
-composite-raster-draw cycle being fairly slow, depending on which exercises you
+composite--raster--draw cycle being fairly slow, depending on which exercises you
```

Comment:
Uhh, does this work right on web? I've been inserting unicode en dashes, but if it works it works.

Reply:
appears to work